### PR TITLE
Fixed font classes, removed logs

### PR DESCRIPTION
**Pull request by erenkan**

### CODE REVIEW

File: .ui/pgVerification.pgx [hunk 1]
```diff
@@ -155,40 +145,40 @@
...
+				"testId": "8B_IHCwxm",
 				"usePageVariable": true
-			},
-			"degree": 2
+			}
 		},
 		{
-			"className": ".sf-label #pgVerification-text",
-			"id": "1880-4a46-ac29-ee4c",
+			"className": ".sf-label #pgVerification-lblResend .verfication.text",
```
Comment (alimoncul) [inline .ui/pgVerification.pgx]: verfication typed wrong

File: themes/mCommerceTheme/styles/pages/PgVerification.json [hunk 2]
```diff
@@ -48,13 +48,7 @@
...
 				"alignSelf": "STRETCH"
 			},
-			"font": {
-				"size": 26,
-				"bold": true,
-				"italic": false,
-				"family": "Nunito",
-				"style": "Bold"
-			},
+			"font": {},
```
Comment (alimoncul) [inline themes/mCommerceTheme/styles/pages/PgVerification.json]: empty object

File: themes/mCommerceTheme/styles/pages/PgVerification.json [hunk 3]
```diff
@@ -68,13 +62,7 @@
...
 				"alignSelf": "STRETCH"
 			},
-			"font": {
-				"size": 16,
-				"bold": true,
-				"italic": false,
-				"family": "Nunito",
-				"style": "Bold"
-			},
+			"font": {},
```
Comment (alimoncul) [inline themes/mCommerceTheme/styles/pages/PgVerification.json]: empty object

**Review by alimoncul — CHANGES_REQUESTED** [review]

File: .ui/pgVerification.pgx (hunk 1 above)
Comment (erenkan) [reply .ui/pgVerification.pgx]: Done.

File: themes/mCommerceTheme/styles/pages/PgVerification.json (hunk 3 above)
Comment (erenkan) [reply themes/mCommerceTheme/styles/pages/PgVerification.json]: Done.

File: themes/mCommerceTheme/styles/pages/PgVerification.json (hunk 2 above)
Comment (erenkan) [reply themes/mCommerceTheme/styles/pages/PgVerification.json]: Done.

**Pull request merged**